### PR TITLE
Fix for the VS 2017 templates.

### PR DESCRIPTION
**Pull request by ILMTitan**
Avoid the Access denied exception that was breaking the templates in VS 2017.

### CODE REVIEW

**Comment by ILMTitan** [comment]
Fix for the first part of Issue #736.

**Comment by ivannaranjo** [comment]
See my comment in the bug. We need to make sure that the templates work in VS 2017 without VS 2015, or the .NET Tools for VS 2015, installed in the system. That is the root cause of the issue. Once I installed the tools in my machines everything worked. But we cannot ask the users to do the same, as it requires having VS 2015 in the system as well.

File: GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/out.project.json [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.4",
-    "Microsoft.AspNetCore.StaticFiles": "1.0.3",
-    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.2",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.1",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.1",
+    "Microsoft.AspNetCore.StaticFiles": "1.1.1",
+    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.1",
     "Microsoft.Extensions.Configuration.Json": "1.0.2",
-    "Microsoft.Extensions.Logging": "1.0.2",
+    "Microsoft.Extensions.Logging": "1.1.1",
```
Comment (ivannaranjo) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/out.project.json]: These seem suspect for a 1.0 project, do they work well with .NET Core 1.0? Have you tried this?

Comment (ILMTitan) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/out.project.json]: I did and it worked fine.

Comment (ivannaranjo) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/out.project.json]: Ok, but is there a reason to update them? Does a comparable 1.0 template from Microsoft do the same thing ?

Comment (ILMTitan) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/out.project.json]: When I tested it, there were some errors regarding out of date dependencies. Not sure how I missed it when I broke them in the first place.

**Pull request merged**